test/integration/find_local_council_test.rb

```diff
@@ -8,6 +8,10 @@ class FindLocalCouncilTest < ActionDispatch::IntegrationTest
 
   context "when visiting the start page" do
     setup do
+      example_generator = GovukSchemas::RandomExample.for_schema("placeholder", schema_type: "frontend")
```

Comment:
I've kept this as a placeholder, because `special_route` formats cannot have a frontend schema. It's on my list to change that. After that I'll update this test. For now this is fine to merge because there wouldn't be an actual difference.